docs/quality-assurance-and-test.md

```diff
@@ -58,3 +162,23 @@ Currently we have templates for:
 
 - Issues
 - Repo structure and standard documentation
+
+When writing technical documentation for features/functionality following topics can be relevant:
+
```

Comment:
Maybe we should link to some good examples within the project in this section?

Reply:
Please do so :)

Reply:
Let's agree on which examples to use at next QA meeting. I suggest [This for Market Roles](https://github.com/Energinet-DataHub/geh-market-roles/blob/main/docs/business-processes/change-of-energy-supplier.md), [This for aggregations](https://github.com/Energinet-DataHub/geh-aggregations/blob/d7750efc6a3c172a0ea69775fa5a157ecd4c9481/source/coordinator/GreenEnergyHub.Aggregation.Application/Coordinator/CoordinatorService.cs#L64).